src/server/capacity_unit_calculator.cpp

```diff
@@ -93,25 +93,36 @@ void capacity_unit_calculator::add_get_cu(int32_t status,
                                           const dsn::blob &value)
 {
     _pfc_get_bytes->add(key.size() + value.size());
-    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
+    if (status == rocksdb::Status::kNotFound) {
+        add_read_cu(1);
+        return;
+    }
+
+    if (status != rocksdb::Status::kOk) {
         return;
     }
```

Comment:
Swap the two judgements, that means return earlier in case of ignore CU calculating, the other cases will calculate CU.

Reply:
`rocksdb::Status::kNotFound` is also `status != rocksdb::Status::kOk`, if swap and happen `rocksdb::Status::kNotFound`, it will not be exected.

Reply:
Use `if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound)`  as what it was to be.

Reply:
> Based on the aliyun([operation cu definition of aliyun](https://help.aliyun.com/document_detail/27304.html?spm=a2c4g.11186623.6.742.451d3354kD2Rb2), [cu calculate example](https://help.aliyun.com/document_detail/27298.html?spm=5176.11065259.1996646101.searchclickresult.41a62ac2iYd8qG)), the way of pegasus to calculate cu is inaccurate that we sometimes ignore the key cu.

As description of pr